search-app/src/cmr/search/services/parameters/conversion.clj

```diff
@@ -198,6 +197,48 @@
    :concept-id :string
    :keyword :keyword})
 
+;; TODO: Generic work: See if we can wrap this in a doseq for all generic types or get from 
+;; a config file.
```

Comment:
Please add to a sub-task or checklist item on the parent ticket and remove the TODO

Reply:
https://bugs.earthdata.nasa.gov/browse/CMR-8621